wp-content/themes/vf-wp/assets/scripts/scripts.js

```diff
@@ -2055,6 +2055,44 @@ function emblBreadcrumbs() {
 })([Element.prototype, Document.prototype, DocumentFragment.prototype]); // Run it on default
 // emblBreadcrumbs();
 
+
+// Configure an object of your locations to detect
```

Comment:
One thing: This JS (and `vfLocationNearest();`) should only run on the Intranet child theme. 

The easiest way to add might be to just insert it directly into the PHP template between some `<script></script>` tags.

Reply:
Ok, back to the drawing board!

Reply:
Updated